src/webhooks/options-automator/options-automator.ts

```diff
@@ -26,15 +26,49 @@ export async function sendOptionAutomatorUpdatesToDataDog(
   message: OptionsAutomatorResponse,
   timestamp: number
 ) {
-  // Not sure how detailed we want the text message to be, but this should be fine initially
-  const params: v1.EventCreateRequest = {
-    title: 'Options Automator Update',
-    text: JSON.stringify(message),
-    alertType: 'info',
-    dateHappened: timestamp,
-    tags: [`sentry_region:${message.region}`],
+  const formatRegionTag = (region: string): string => {
+    if (region === 'us' || region === 'de') {
+      return `sentry_region:${region}`;
+    } else {
+      return `sentry_region:st-${region}`;
+    }
+  };
+
+  const formatAlertType = (optionType: string): v1.EventAlertType => {
+    return optionType === 'updated_options' ||
+      optionType === 'set_options' ||
+      optionType === 'unset_options'
+      ? 'success'
+      : 'error';
   };
-  await DATADOG_API_INSTANCE.createEvent({ body: params });
+
+  for (const optionType in message) {
+    if (optionType === 'region') continue;
+    for (const option of message[optionType]) {
+      const text = {
+        change: optionType,
+        option: option,
+      };
+
+      const region = formatRegionTag(message.region);
+
+      const alertType = formatAlertType(optionType);
+
+      const params: v1.EventCreateRequest = {
+        title: 'Options Automator Update',
+        text: JSON.stringify(text),
+        alertType: alertType,
+        dateHappened: timestamp,
+        tags: [
+          region,
+          `source_tool:options-automator`,
```

Comment:
Since now we're sending an event per option changed, is it useful to have the option name as a tag?